### PR TITLE
Playbook for create vm

### LINKED ISSUE #21: Convert create-vm shell script to ansible playbook

### PR DESCRIPTION
**Pull request by trishnaguha**
Fixes #21 
I will push the change for README file to the same branch (In this PR).

### CODE REVIEW

File: create-vm.yml [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+      register: libvirtd
+
+    - name: Wait for libvirtd to start
+      wait_for:
+        timeout: 30
+      when: libvirtd.changed
+
+    - name: Create domain {{ domain }}
+      command:
+        echo 'Empty message'
```
Comment (gbraad) [inline create-vm.yml]: ?

Comment (trishnaguha) [reply create-vm.yml]: I wanted to display only the message `Create domain {{ domain }}`. Not sure how to do **nothing** in the play. Just like we do `pass` in Python.

**Review by gbraad — APPROVED** [review]
Nice...

File: create-vm.yml [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+  file: path={{ item }} state=touch
+  with_items:
+    - '{{ tmpdir.stdout }}/meta-data'
+    - '{{ tmpdir.stdout }}/user-data'
+
+- name: cloud-init meta-data
+  lineinfile: dest='{{ tmpdir.stdout }}/meta-data' line='instance-id{{':'}} "{{ domain }}"\nlocal-hostname{{':'}} "{{ domain }}"'
+
+- name: cloud-init user-data
+  lineinfile: dest='{{ tmpdir.stdout }}/user-data' line='#cloud-config\nuser{{':'}} atomic-user\npassword{{':'}} atomic\nchpasswd{{':'}} {expire{{':'}} False}\nssh_pwauth{{':'}} True'
```
Comment (gbraad) [inline create-vm.yml:33]: Can you improve the merge/change now, by also moving this into a template?

Comment (trishnaguha) [reply create-vm.yml:33]: OK.

**Pull request merged**